ExcelReader/wwwroot/js/Core.js

```diff
@@ -0,0 +1,67 @@
+﻿
+$(document).ready(() => {
+    console.log("document is ready")
+    var loadDataBttn = $("#loadTableData");
+    let headers;
+    // Assuming you have a JWT token stored in a variable called 'jwtToken'
+    let jwtToken = localStorage.getItem('jwtToken');
+    if (jwtToken) {
+        // Include the token in the request headers
+        headers = {
+            'Authorization': 'Bearer ' + jwtToken
+        };
+    }
+    loadDataBttn.click(() => {
+        $.ajax({
+            url: "http://localhost:6769/it-request",
```

Comment:
Create configuration or constant for base URL and use that every where.

Reply:
done